perspective-view.framer/framer/backups/backup-1460224356.coffee

```diff
@@ -1,8 +1,8 @@
 # Perspective module
 
-{PerspectiveLayer} = require "perspective-view"
+{Perspective} = require "perspective-view"
 
-myPerspective = new PerspectiveLayer
+p = new Perspective
```

Comment:
Yep - committed new version with class "PerspectiveView" and variable name "perspectiveView".